### PR TITLE
update modules path in the reference file

### PR DESCRIPTION
**Pull request by kostasb**
modules reside under the modules.d directory
metricbeat.yml correctly sets path: ${path.config}/modules.d/*.yml
the reference file was still using conf.d instead of modules.d, which this PR fixes
 
followed @ruflin 's recommendations in elastic/beats#11491

### CODE REVIEW

**Comment by ruflin** [comment]
@kostasb Thanks. Could you run `make fmt update` on your end and push the diff? Somehow CI is not happy with it yet.

File: metricbeat/module/elasticsearch/elasticsearch_integration_test.go [hunk 1]
```diff
@@ -146,9 +146,9 @@ func getEnvPort() string {
 // GetConfig returns config for elasticsearch module
 func getConfig(metricset string) map[string]interface{} {
 	return map[string]interface{}{
-		"module":                     elasticsearch.ModuleName,
-		"metricsets":                 []string{metricset},
-		"hosts":                      []string{getEnvHost() + ":" + getEnvPort()},
+		"module":     elasticsearch.ModuleName,
```
Comment (ruflin) [inline metricbeat/module/elasticsearch/elasticsearch_integration_test.go]: That diff should not be here. What golang version do you have locally? Make sure you are on 1.11.(5)

Comment (kostasb) [reply metricbeat/module/elasticsearch/elasticsearch_integration_test.go]: I have fmt 5.3.0 on the host, and go was 1.10.X
I installed go 1.11.5, run fmt and will push again

**Comment by ruflin** [comment]
Interestingly this complains about the config file in the x-pack directory. Probably you have to run also `make update` in `x-pack/metricbeat`. This is something we should fix on our end to not have to do it manually.

Alternative is to run `make update` in the top directory but will take a bit.

**Comment by ycombinator** [comment]
For consistency I think we should also change all other occurrences of `conf.d/*.yml` under `*.config.modules` in the beat repo:

```
$ git grep '${path.config}/conf.d/'
auditbeat/_meta/common.reference.yml:  path: ${path.config}/conf.d/*.yml
auditbeat/auditbeat.reference.yml:  path: ${path.config}/conf.d/*.yml
auditbeat/docs/reload-configuration.asciidoc:  path: ${path.config}/conf.d/*.yml
metricbeat/_meta/common.reference.yml:  path: ${path.config}/conf.d/*.yml
metricbeat/metricbeat.reference.yml:  path: ${path.config}/conf.d/*.yml
x-pack/auditbeat/auditbeat.reference.yml:  path: ${path.config}/conf.d/*.yml
x-pack/metricbeat/metricbeat.reference.yml:  path: ${path.config}/conf.d/*.yml
```

**Comment by kostasb** [comment]
Agreed, although auditbeat doesn't seem to use the modules directory, the latest commit aligns all references in the repo to use modules.d instead of conf.d

**Comment by kaiyan-sheng** [comment]
@kostasb Sorry for the delay on this PR. Could you rebase this PR so the CI will pass?

**Pull request closed without merging**